client/App.jsx

```diff
@@ -4,11 +4,14 @@ import { Provider } from 'react-redux';
 import { PersistGate } from 'redux-persist/integration/react';
 import { Router } from './Router';
 import { store, persistor } from './store/store';
+import { AuthProvider } from './store/users/AuthContext';
 
 export const App = () => (
-  <Provider store={store}>
-    <PersistGate loading={null} persistor={persistor}>
-      <Router />
-    </PersistGate>
-  </Provider>
+  <AuthProvider>
```

Comment:
Interesting. Ok so `AuthProvider` sits outside the store. Can it still use the store?

Reply:
Whoops, I swapped it to move outside `<Router />`, especially since we don't actually use the store at the moment - but we can just put it inside the `<PersistGate />` when we need to